shenyu-admin/src/main/resources/application.yml

```diff
@@ -88,6 +88,7 @@ shenyu:
 #      namespace: 1c10d748-af86-43b9-8265-75f487d20c6c
 #      username:
 #      password:
+#      context-path:
```

Comment:
default context path is nacos?

Reply:
yes, in nacos-client, it will be set to 'nacos' when the contextPath property value is empty.